truffle/com.oracle.truffle.dsl.processor/src/com/oracle/truffle/dsl/processor/InstrumentRegistrationProcessor.java

```diff
@@ -57,24 +57,34 @@ public SourceVersion getSupportedSourceVersion() {
         return SourceVersion.latest();
     }
 
+    private static final int NUMBER_OF_PROPERTIES_PER_ENTRY = 4;
```

Comment:
Ok, assuming it's unlikely to change any time soon, we can change this code later if need be.